docs/trace_collection.md

```diff
@@ -121,7 +120,7 @@ If you want to trace your OkHttp requests, you can add the provided [Interceptor
 
 ```kotlin
 val okHttpClient =  OkHttpClient.Builder()
-    .addInterceptor(TracingInterceptor())
+    .addInterceptor(DatadogRumMonitor())
```

Comment:
```suggestion
    .addInterceptor(RumInterceptor())
```

Reply:
Again, no, the recommended way is to use the combined interceptor aka DatadogInterceptor

Reply:
Yup but you should have `DatadogInterceptor` there not `DatadogRumMonitor` then .